# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/ActorsParent.cpp
# Commit: 8726e5de9204
# Full Hash: 8726e5de920473058ffc6df94633a0263e31decd
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-12-11 21:46:29
# Regressor Bug: 1600906
# File Overlap Count: 1
# Description:
#   Bug 1600906 - Added const in UpdateRefcount. r=dom-workers-and-storage-reviewers,ytausky
#   
#   Depends on D56008
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D56009
# ==============================================================================

diff -r ac4d8e02e943 -r 8726e5de9204 dom/indexedDB/ActorsParent.cpp
--- a/dom/indexedDB/ActorsParent.cpp	Wed Dec 11 14:45:40 2019 +0000
+++ b/dom/indexedDB/ActorsParent.cpp	Wed Dec 11 14:40:38 2019 +0000
@@ -4915,7 +4915,7 @@
   CachedStatement mSelectStatement;
   CachedStatement mInsertStatement;
 
-  UpdateRefcountFunction* mFunction;
+  UpdateRefcountFunction* const mFunction;
 
   nsresult mErrorCode;
 
